config/routes.rb

```diff
@@ -1,3 +1,7 @@
-Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+Rails.application.routes.draw do  
+  resources :houses, only: [:show, :index]  
+  resources :users, only: [:create, :index]
+  resources :favourites
+  post 'authenticate', to: 'authentication#authenticate'
```

Comment:
Let's make sure our route names are RESTful.

Reply:
Let's please address this :) Forgot to remind you in my 2nd review

Reply:
Our routes still are not fully RESTful. While it's not a rule (more of a generally accepted convention) and I won't push it further, it's a high recommendation that we get used to using RESTful names for our routing. I suggest reading articles about REST and REST API's to master the principle